src/main/java/org/opensearch/timeseries/ratelimit/ColdStartWorker.java

```diff
@@ -174,6 +177,18 @@ protected void executeRequest(FeatureRequest coldStartRequest, ActionListener<Vo
                                         : new ParameterizedMessage("Failed to load model {}.", modelState.getModelId())
                                 );
                         }
+                    } else {
+                        // run once scenario
+                        String taskId = coldStartRequest.getTaskId();
+                        if (taskId != null) {
+                            Map<String, Object> updatedFields = new HashMap<>();
+                            updatedFields.put(TimeSeriesTask.STATE_FIELD, TaskState.INACTIVE.name());
```

Comment:
why is it always being set to inactive here, a little confused on the run once scenario here

Reply:
This branch is for the cold start failure where we don't have enough data. Since it is run once, we won't retry as we did in real time.

Reply:
According to the state transition graph from the overview section, we need to use INACTIVE.